### ISSUE TITLE
Tabs

### ISSUE THREAD

**Issue by halimahLDN** [Greater-London-Authority/ldn-viz-tools#759]
- [x] Insufficient colour contrast
![image](https://github.com/user-attachments/assets/32df07d1-8cae-4092-9921-63af5efde656)
- [ ] Screenreader says all of the tabs are `selected` instead of the single one that's highlighted
